examples/loader-datawrapper/src/index.md

```diff
@@ -0,0 +1,83 @@
+# Datawrapper data loader
+
+Here’s a Python data loader that generates an HTML [iframe](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe) tag that embeds a [Datawrapper](https://www.datawrapper.de/) chart.
+
+```python
+import sys
+
+import pandas as pd
+from datawrapper import Datawrapper
+
+# Get an extract of arrests made by the Baltimore Police Department
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/palewire/first-automated-chart/main/_notebooks/arrests.csv",
+    parse_dates=["ArrestDateTime"]
+)
+
+# Create a year column
+df['year'] = df.ArrestDateTime.dt.year
+
+# Calculate the total number by year
+totals_by_year = df.year.value_counts().sort_index().reset_index()
+
+# Connect to the datawrapper api
+dw = Datawrapper()
+
+# Create a chart
+chart_config = dw.create_chart(
+    # Headline the chart
+    title="Baltimore Arrests",
+    # Set the type
+    chart_type="column-chart",
+    # Give the data
+    data=totals_by_year,
+    # Configure other bits
+    metadata={
+        # Set the description
+        "describe": {
+            "source-name": "OpenBaltimore",
+            "source-url": "https://data.baltimorecity.gov/datasets/baltimore::bpd-arrests/about",
+        }
+    }
+)
+
+# Pull out the chart's unique identifier
+chart_id = chart_config["id"]
+
+# Publish the chart
+dw.publish_chart(chart_id)
+
+# Write the chart's embed code to stdout
+html = dw.get_iframe_code(chart_id, responsive=True)
+sys.stdout.write(html)
+```
+
+<div class="note">
+
+To run this data loader, you will need to create an API token in Datawrapper and set it as an environment variable named `DATAWRAPPER_ACCESS_TOKEN`.
```

Comment:
For the **DATAWRAPPER_ACCESS_TOKEN** can we specify where to create it (https://app.datawrapper.de/account/api-tokens) and the scopes we have to create? I initially tried with a narrow scope (Chart | read  write) and the data loader created the chart, but couldn't read it back. It worked when I enabled more reading capabilities (Folder | read  write; Theme | read; User | read; Visualization | read), but I'm not sure which of those were strictly necessary.

It could also be useful to explain how to set it inside the venv.

Reply:
I'm open to whatever environment management technique you want to recommend. I typically use a `.env` file with the standard pipenv conventions that is excluded from the git repository. In this case, for simplicity's sake I just did something like:

```bash
DATAWRAPPER_ACCESS_TOKEN=xxxyyzzz pipenv run npm run dev
```